lib/action_cable/channel/broadcasting.rb

```diff
@@ -0,0 +1,25 @@
+module ActionCable
+  module Channel
+    module Broadcasting
+
+      # Broadcast a hash directly to a named <tt>broadcasting</tt>. It'll automatically be JSON encoded.
+      def broadcast(broadcasting, message)
```

Comment:
I don't think this is really needed? It doesn't add anything over ActionCable.server.broadcast. Let's nix it.